wpilibj/src/main/java/edu/wpi/first/wpilibj/smartdashboard/SendableBuilderImpl.java

```diff
@@ -61,8 +61,8 @@ void stopListener() {
   private boolean m_actuator;
 
   /**
-   * Set the network table.  Must be called prior to any Add* functions being
-   * called.
+   * Set the network table.  Must be called prior to any Add* functions being called.
+   *
```

Comment:
I understand this is probably due to editor auto-formatting, but the quantity of whitespace-only changes here is a bit distressing, as they dwarf the functional changes.  Can these be separated out to a different PR?

Reply:
It's kind of hard to revert only the whitespace changes at this point; I can create a separate PR with *only* the whitespace changes, though.

Reply:
I mean, you could just look at the diff and do the opposite of what it says by hand.